### ISSUE TITLE
go test ./... failed

### ISSUE THREAD

**Issue by hliu1020** [ava-labs/avalanche-network-runner#81]
OS: macOS Moneterey
golang: go version go1.17.6 darwin/amd64
----

▶ go test ./...
\# gopkg.in/olebedev/go-duktape.v3
In file included from _cgo_export.c:4:
debugger.go:23:13: warning: unused function '_duk_debugger_attach' [-Wunused-function]
\# github.com/ava-labs/coreth/plugin/evm
../../go/pkg/mod/github.com/ava-labs/coreth@v0.8.1-rc.0/plugin/evm/vm.go:86:2: cannot use &VM{} (type *VM) as type block.ChainVM in assignment:
	*VM does not implement block.ChainVM (wrong type for Connected method)
		have Connected(ids.ShortID) error
		want Connected(ids.ShortID, version.Application) error
FAIL	github.com/ava-labs/avalanche-network-runner/k8s [build failed]
FAIL	github.com/ava-labs/avalanche-network-runner/local [build failed]
FAIL	github.com/ava-labs/avalanche-network-runner/network [build failed]
ok  	github.com/ava-labs/avalanche-network-runner/utils	(cached)

**Comment by danlaine** [comment]
@hliu1020 What commit are you on? `git rev-parse HEAD`

**Comment by hliu1020** [comment]
I'm using the latest code. Just git clone then run `go test ./...`

git rev-parse HEAD:
6713414330d18a70358c167bed7439c02d2f2465

I have tried to use docker to run local five nodes and that succeed. When  using docker, I came into an issue that the local node only can listen at http host: 127.0.0.1 and it's inconvenient to connect from host to the container if the avalanchego node listening only at host 127.0.0.1 inside container(I have not found any way to configure this flag `http-host`, maybe I missed something?). So I modified a little so that can configure httpHost to `0.0.0.0` (not 127.0.0.1) similar as `config.HTTPPortKey` and finally can using tools like hardhat to connect the five local nodes running inside container.

Can the flag `HTTPHostKey` be configurable? I have search the code and not found any way. Any hints are appreciated.
